rules/__tests__/missing_expect_assertions_call_test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rules = require('../..').rules;
+
+const ruleTester = new RuleTester();
+const expectedMsg =
+  'Every test should have expect.assertions({number of assertions}) OR expect.hasAssertions() as first expression';
```

Comment:
I think this might read better as

    Every test should have either `expect.assertions(<number of assertions>)` or `expect.hasAssertions()` as it's first expression